docs/data/date-pickers/time-picker/time-picker.md

```diff
@@ -19,18 +19,14 @@ They will be renamed at the end of the v6 beta phase to have the same name as th
 (`NextTimePicker` will become `TimePicker`, ...)
 :::
 
-The selected time is indicated by the filled circle at the end of the clock hand.
```

Comment:
Do you think this bit of information is useful ?

Reply:
IMHO, it's somewhat redundant. The result is pretty evident from the demos. 🤔